src/qibolab/qubits.py

```diff
@@ -148,21 +95,9 @@ class QubitPair:
     Acts as target on two-qubit gates.
     """
 
-    gate_fidelity: float = 0.0
-    """Gate fidelity from standard 2q RB."""
-
-    cz_fidelity: float = 0.0
-    """Gate fidelity from CZ interleaved RB."""
-
-    coupler: Optional[Qubit] = None
+    # coupler: Optional[Qubit] = None
+    # FIXME: I think this is not needed but not sure yet
+    # This information is not provided in the runcard so it is not
+    # parsed by serialize.py
```

Comment:
I also believe it can be dropped. The connection is made by the channel used for the coupler in the `TwoQubitNatives`, we should not need anything more than that.

Reply:
I am not sure if it will be needed at some point but for now tests are passing so I removed in 06b654650a7fce178fb45d18eddc450c38b755c1.